README.md

```diff
@@ -160,7 +160,7 @@ Clients are available for the following destinations:
 * AppOptics - https://github.com/ysamlan/go-metrics-appoptics
 * Librato - https://github.com/mihasya/go-metrics-librato
 * Graphite - https://github.com/cyberdelia/go-metrics-graphite
-* InfluxDB - https://github.com/vrischmann/go-metrics-influxdb
+* InfluxDB - https://github.com/vrischmann/go-metrics-influxdb / https://github.com/tehsphinx/metrics
```

Comment:
Can I suggest naming your repo something more descriptive? `go-metrics-influxdb-alt` or something along those lines? I know, I know, naming is hard, but I feel like just calling it "metrics" is asking for trouble down the lines

Reply:
sounds reasonable. I will try to think of something. Problem ist that the name should also be the package name.

Reply:
What do you think of `fluxmetrics`?